tests/unit/components/search-autocomplete/SearchAutocomplete.spec.jsx

```diff
@@ -17,41 +16,78 @@ describe('SearchAutocomplete', () => {
         textFieldProps={{ label: 'Search' }}
       />
     )
-  })
 
-  it('renders autocomplete with search input', () => {
     const searchInput = screen.getByLabelText('Search')
     expect(searchInput).toBeInTheDocument()
   })
 
+  it('renders autocomplete with search input value', () => {
+    render(
+      <SearchAutocomplete
+        options={options}
+        search='France'
+        setSearch={setSearch}
+        textFieldProps={{ label: 'Search' }}
+      />
+    )
+
+    const searchInput = screen.getByLabelText('Search')
+    expect(searchInput.value).toBe('France')
+  })
+
   it('updates search input on typing', async () => {
-    const user = userEvent.setup()
+    render(
+      <SearchAutocomplete
+        options={options}
+        search=''
+        setSearch={setSearch}
+        textFieldProps={{ label: 'Search' }}
+      />
+    )
 
     const searchInput = screen.getByLabelText('Search')
 
-    await waitForTimeout(() => user.type(searchInput, 'Finland'))
+    await act(async () => {
```

Comment:
why you removed waitForTimeout?

Reply:
i return old test, now its passing, before somehow not.